src/components/ui/SideNavigation/_partials/Sub.tsx

```diff
@@ -34,8 +34,8 @@ interface subLinkProps {
  */
 export default function SubLink({ item, activeLinkStyle, subitem }: subLinkProps): ReactElement {
   return (
-    <li className="relative mt-4 text-sm text-gray-500" style={activeLinkStyle}>
-      <Link href={{ pathname: item.link, hash: subitem.link }} className={`relative text-gray-500 opacity-50 hover:opacity-100 ${!subitem.exact ? "activable-link" : ""}`}>
+    <li data-testid="subId" className="relative mt-4 text-sm text-gray-500" style={activeLinkStyle}>
+      <Link data-testid="linkId" href={{ pathname: item.link, hash: subitem.link }} className={`relative text-gray-500 opacity-50 hover:opacity-100 ${!subitem.exact ? "activable-link" : ""}`}>
```

Comment:
I wonder, don't we have this already tested? because if it's tested then we don't need to test it again.